spam/03_spam_data_slicing_tutorial.py

```diff
@@ -352,93 +239,166 @@ def textblob_polarity(x):
 sfs = [short_link] + extra_sfs
 slice_names = [sf.name for sf in sfs]
 
+# %% [markdown]
+# Let's see how the `sklearn` model we learned before performs on these new slices!
+
 # %%
 applier = PandasSFApplier(sfs)
 S_test = applier.apply(df_test)
 
-slice_scorer = SliceScorer(scorer, slice_names)
-slice_scorer.score(
+slice_scorer = SliceScorer(slice_names, metrics=["accuracy", "f1"])
+slice_scorer.score_slices(
     S=S_test, golds=Y_test, preds=preds_test, probs=probs_test, as_dataframe=True
 )
 
 # %% [markdown]
-# ## 5. Improve slice performance
+# Looks like they do well — we'll want to monitor these to make sure performance changes on one don't hurt another.
```

Comment:
Would change to something like: "Looks like some do extremely well on our small test set, while others do decently; at very least, we may want to monitor these to make sure that as we iterate to improve certain slices like `short_link`, we don't hurt the performance of others.  Next, we'll introduce a model that helps us to do this balancing act automatically!"